src/testUtils.js

```diff
@@ -47,25 +66,10 @@ function runTest(dirName, transformName, options, testFilePrefix) {
   );
   // Assumes transform is one level up from __tests__ directory
   const module = require(path.join(dirName, '..', transformName + '.js'));
-  // Handle ES6 modules using default export for the transform
-  const transform = module.default ? module.default : module;
-
-  // Jest resets the module registry after each test, so we need to always get
```

Comment:
It looks like these comments have gone missing - Could you please re-insert them?